### PR TITLE
Recipe App

### PR DESCRIPTION
**Pull request by AlaaAlsalem**
Here are the things we created with this repo
> Initialize project with ruby on rails
> Create a migration database
> Create modules 
> Create controllers 
> Create and link views with controllers
> Add Authentications, and authorizations
> Style the site so it looks friendly and easy to use  for the end user

### CODE REVIEW

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @AlaaAlsalem, 

Good job so far! 🔥
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes ♻
- [ ] Let's write Unit and Integration tests!

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- ♻ Improve your auth pages styles

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/models/food_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Food, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (rotimiazeez) [inline spec/models/food_spec.rb]: - Kindly add some tests to your food model 👍🏽

File: spec/models/recipe_food_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe RecipeFood, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (rotimiazeez) [inline spec/models/recipe_food_spec.rb]: - Kindly add some tests to your recipe_food model 👍🏽

File: spec/models/recipe_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (rotimiazeez) [inline spec/models/recipe_spec.rb]: - Kindly add some tests to your recipe model 👍🏽

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```
Comment (rotimiazeez) [inline spec/models/user_spec.rb]: - Kindly add some tests to your user model 👍🏽

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi TEAM 👍🏽, 

## Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by DammyShittu — APPROVED** [review]
Hi @AlaaAlsalem @BouaikIlham,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**